pkg/rtc/room.go

```diff
@@ -296,9 +296,17 @@ func (r *Room) CloseIfEmpty() {
 
 	r.lock.RLock()
 	numParticipants := len(r.participants)
+	if numParticipants == 1 {
```

Comment:
what if there are multiple participants here?

Reply:
then we won't close the room

Reply:
we should close the room as long as there are no visible participants left.